### ISSUE TITLE
Bookmarks navigation doesn't work.

### ISSUE THREAD

**Issue by lamuswawir** [syncfusion/flutter-widgets#1048]
Hello, I have added bookmark navigation, though the bookmarks are displayed, a few work on clicking them. Opens up child bookmarks but doesn't navigate to the destination on clicking the bookmark.

This is the document.

[Uganda-Clinical-Guidelines-2016.pdf](https://github.com/syncfusion/flutter-widgets/files/10486528/Uganda-Clinical-Guidelines-2016.pdf)

**Comment by irfanajaffer** [comment]
we have checked through the entire pdf and bookmarks are properly navigating to its destination. can you please confirm , which section of pages you are experiencing this issue?

To get more accurate results, we request you to add the below code snippet on your end:
`  //Creates document bookmark
  PdfBookmark bookmark = document.bookmarks.add(text);  //Create a destination with page and location  PdfDestination destination = PdfDestination(page, offset);  //Set the destination mode  destination.mode = PdfDestinationMode.location;
  //Sets the destination
  bookmark.destination = destination;`

follow the below links for more information,
https://help.syncfusion.com/flutter/pdf/working-with-bookmarks

**Comment by lamuswawir** [comment]
Thank you, it is a problem with the document, tested it with other documents and it works fine.

**Comment by irfanajaffer** [comment]
Okay, can you please share the steps to replicate the reported issue? So that we can assist with you further in this

**Comment by lamuswawir** [comment]
I added opening bookmarks with this code
```
final GlobalKey<SfPdfViewerState> _pdfViewerKey = GlobalKey();

@override
Widget build(BuildContext context) {
  return Scaffold(
    appBar: AppBar(
      title: Text(bookTitle),
      actions: <Widget>[
        IconButton(
          icon: Icon(
            Icons. bookmark,
            color: Colors.white,
          ),
          onPressed: () {
            _pdfViewerKey.currentState?.openBookmarkView();
          },
        ),
      ],
    ),
    body: SfPdfViewer.file(
      File(filePath),
      key: _pdfViewerKey,
    ),
  );
}
```
When I further examined the destinations of the bookmarks which were not working, I found that the destination was null, I assumed it was the PDFs with a problem as other PDFs were navigating via bookmarks.

**Comment by deepika134** [comment]
We tried to reproduce the reported issue with provided details, but the bookmark navigation works fine on our side. We have prepared a sample for your reference, which can be downloaded from the attachment. If you still can reproduce the reported issue in your use case, then kindly share the modified issue reproducing sample which will be helpful for us to provide a prompt solution at the earliest.
[github_1048.zip](https://github.com/syncfusion/flutter-widgets/files/10538514/github_1048.zip)
